task-management/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -316,6 +318,13 @@
                 this.menuActions = this.menuActions.filter(menuAction => menuAction.enabled);
               });
             }
+          } else if (event.detail.name === 'noProject') {
+            this.menuActions = [];
+            this.enableDelete = event.detail.task.createdBy === eXo.env.portal.userName ? true : false;
```

Comment:
No need for the part after the '?' Charater right?

Reply:
It is a backport PR, I just copy and paste of the code for the 6.1